packages/tauri/src-tauri/Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 name = "Padloc"
```

Comment:
Let's change this to lowercase, to get rid of that warning during the build process.

Reply:
I considered it, but it makes the menu/window name in macOS show lowercase, which is inconsistent with their guidelines. I prefer to see that warning instead, but can change it!

Reply:
I think that's only if you're running it in dev mode. Once bundled the name in the app menu should be uppercase. Can you verify that?

Reply:
I'll change and verify this later when I pull up my macbook again, if that's fine.